td-agent/apt/install-test.sh

```diff
@@ -31,6 +31,10 @@ case ${code_name} in
 	apt install -V -y piuparts mount gnupg curl eatmydata
 	gpg_command=gpg
 	;;
+    focal)
+	DEBIAN_FRONTEND=noninteractive apt install -V -y piuparts mount gnupg1 curl eatmydata
```

Comment:
I want to add `DEBIAN_FRONTEND=noninteractive` to all versions and merge it to `*)` block if it's harmless.

Reply:
I'll check it again whether it is focal specific.

Reply:
I've unified into `*)` block.
@ashie